app/_src/kic-v2/deployment/kong-enterprise.md

```diff
@@ -1,122 +1,101 @@
-kong-validation-webhook   ClusterIP      10.63.240.154   <none>          443/TCP                      4m34s
-postgres                  ClusterIP      10.63.241.104   <none>          5432/TCP                     4m34s
-
-```
-
-> Note: Depending on the Kubernetes distribution you are using, you might or might
-not see an external IP assigned to the three `LoadBalancer` type services. Please see
-your provider's guide on obtaining an IP address for a Kubernetes Service of
-type `LoadBalancer`. If you are running minikube, an external IP address may not be listed.
-For more information about accessing the service of type `LoadBalancer` in minikube,
-see [LoadBalancer access](https://minikube.sigs.k8s.io/docs/handbook/accessing/#loadbalancer-access).
-
-### Setup Kong Manager
-
-Next, if you browse to the IP address or host of the `kong-manager` service in your Browser,
-which in our case is `http://34.83.242.237`.
-Kong Manager should load in your browser.
-Try logging in to the Manager with the username `kong_admin`
-and the password you supplied in the prerequisite, it should fail.
-The reason being we've not yet told Kong Manager where it can find the Admin API.
-
-Let's set that up. We will take the External IP address of `kong-admin` service and
-set the environment variable `KONG_ADMIN_API_URI`:
-
-```bash
-KONG_ADMIN_IP=$(kubectl get svc -n kong kong-admin --output=jsonpath='{.status.loadBalancer.ingress[0].ip}')
-kubectl patch deployment -n kong ingress-kong -p "{\"spec\": { \"template\" : { \"spec\" : {\"containers\":[{\"name\":\"proxy\",\"env\": [{ \"name\" : \"KONG_ADMIN_API_URI\", \"value\": \"${KONG_ADMIN_IP}\" }]}]}}}}"
-```
-
-It will take a few minutes to roll out the updated deployment and once the new
-`ingress-kong` pod is up and running, you should be able to log into the Kong Manager UI.
-
-As you follow along with other guides on how to use your newly deployed the {{site.kic_product_name}},
-you will be able to browse Kong Manager and see changes reflected in the UI as Kong's
-configuration changes.
-
-## Using Kong for Kubernetes with Kong Enterprise
-
-Let's setup an environment variable to hold the IP address of `kong-proxy` service:
-
-```bash
-$ export PROXY_IP=$(kubectl get -o jsonpath="{.status.loadBalancer.ingress[0].ip}" service -n kong kong-proxy)
-```
-
-Once you've installed Kong for Kubernetes Enterprise, please follow our
-[getting started](/kubernetes-ingress-controller/{{page.kong_version}}/guides/getting-started) tutorial to learn more.
+## Before you begin
+* Create the `kong` namespace.
+    ```bash
+    $ kubectl create namespace kong
+    ```
+    The results should look like this:
+    ```text
+    namespace/kong created
+    ```
+
+* Create Kong Enterprise bootstrap password. Replace `<cloudnative>` with a random password of your choice and note it down
+    ```bash
+    $ kubectl create secret generic kong-enterprise-superuser-password  -n kong --from-literal=password=<cloudnative>
```

Comment:
Using `<cloudnative>` means that I can't copy/paste when testing. Can we go back to using `cloudnative`?

Reply:
Fixed